src/main/java/bio/terra/cloudfiletodatastore/deltalayer/model/PointCorrection.java

```diff
@@ -0,0 +1,34 @@
+package bio.terra.cloudfiletodatastore.deltalayer.model;
+
+import java.util.List;
+
+public class PointCorrection {
+
+  String name;
```

Comment:
nit: Maybe change the property `name` to `dataSet` like how it is in `BqInserts`?